lib/rules/max-empty-lines/index.js

```diff
@@ -23,47 +21,28 @@ const rule = function (max) {
     if (!validOptions) {
       return
     }
-
+    
+    // `//`-comments are need to be processed by postcss-scss
+    // To do that we have to use .toString()
     const rootString = root.toString()
```

Comment:
I guess I couldn't tell properly. Point is if I use `root.source.input.css`, line counts will be wrong on scss comments. And to fix, need to do extra work.